### ISSUE TITLE
Script Won't work at all

### ISSUE THREAD

**Issue by thatmanwhousesgithub** [Valokoodari/btd6-ahk#56]
I have installed everything, used the hot key, but the script is unresponsive, nothing happens.

**Comment by kwalk123** [comment]
The first thing you would need to do is edit the main script to use the halloween event files. But even with that the files no longer match the game, and therefore the script won't work. I found I had to reduce the ImageSearch difference allowance parameter just to get it to recognize play status (I modified the script to use more rewarding Impoppable maps and got rid of the timer that interfered with multi-tasking). But I understand doing so would cause the script to not work for some people.

No matter what I set the parameter to and which graphics I tried, it failed to recognize the two maps that I tried. There appears to be a lot more animation (rain, fog, etc.) that throws off ImageSearch. This could be solved by storing the selected map from the map selection screen, which would get rid of the need to detect with ImageSearch, but who knows what issues that might cause or that would remain. I got the in game automation to work again, but everything is manual after victory. Good enough for me, but not great.
